### ISSUE TITLE
Provide bright line to separate modules in right hand panel in darkroom

### ISSUE THREAD

**Issue by LateJunction** [darktable-org/darktable#10858]
**Is your feature request related to a problem? Please describe.**
I have a 4K monitor that is only 27" wide; pixels are consequently quite small (compared to what I have historically used). When coupled with somewhat degraded eyesight, this results in some eye-ball scanning being required to distinguish between open modules in the darkroom view - especially as some modules are quite 'long' when the masking options are being used.

**Describe the solution you'd like**
This difficulty would be eliminated if there was a one- or two-pixel width horizontal ruled line either at the bottom of an module (expanded or not) or at the top of the module. It would have to be of a suitable brightness to stand out. For the default theme, at least, a coloured line would make a welcome addition.

**Alternatives**

**Additional context**

**Comment by Nilvus** [comment]
There's already such line, a subtle one and it's wanted. And this had been discussed and will not be brighter. Or maybe you could provide a mockup in case you have a better idea AND one that is, visible of course, but not to visible regardless of other parts of the UI. Another thing: 4k let thing really small but you could also increase things with few tweaks (that are here for that):
- use scale factor of your screen (display options in quite all major OS) and use 200% one
- increase fonts (not the best option as it only increase fonts could help anyway
- in darktable main settings (and general tab), use option "GUI controls and text DPI" and increase value to have less smaller UI in darktable. The best option to use if you want to increase UI elements only in darktable app. You could also use specific font size for darktable if needed too

**Comment by LateJunction** [comment]
Thanks for this useful guidance. Perhaps it is a measure of how effective the existing "bright" line is, that I never knew it existed until you mentioned it and I had moved my eye to within 15 cm of my new monitor or 10 cm of my old one! Is there any possibility the brightness of this line could be made a variable option ( 'default', 'brighter') within, say, the 'general' section of darktable preferences?

**Comment by LateJunction** [comment]
Replied by posting on github

**Comment by Nilvus** [comment]
> Thanks for this useful guidance. Perhaps it is a measure of how effective the existing "bright" line is, that I never knew it existed until you mentioned it and I had moved my eye to within 15 cm of my new monitor or 10 cm of my old one! Is there any possibility the brightness of this line could be made a variable option ( 'default', 'brighter') within, say, the 'general' section of darktable preferences?

darktable already have a lot of options and we agree, in darktable team, that there is already quite too much. So we won't add an option for such a minor thing. I use grey theme and it's easily visible on my side (and you're the first to ask for that, and actual UI is already 2 years old), so if you have such difficulties, I think this is not the issue, but how your screen is displayed (what I have posted in my first answer and maybe how it's calibrated too).

That said, I realize that you don't precise which theme you use? Did you test with other theme too if you have same visibility or if it's different on your side depending on theme? Thus, it would be good if you could post a screenshot of modules panel to see what you have on your side.

**Comment by LateJunction** [comment]
Yes, I'll provide screen shots later. I am using the default ('darktable') theme as it is the most effective for me.

**Comment by Nilvus** [comment]
I check this and unfortunately, I think this will be a won't fix. I tested and this will change other things and make the Ui clearer and so nearer than dark theme. And this will not be logical regarding other themes I think.

You could test what I mean by copy/paste following line in CSS editor in general tab of darktable main settings and save and apply it. You will see that line is brighter, but not only line. And that change the theme. If the rendering is good for you, I suggest so to keep that line and use it for a better view on your side:

`@define-color plugin_bg_color @grey_25;` (by default, it's grey_20, so 5% darker).
